crates/gen/src/com_interface.rs

```diff
@@ -28,21 +30,43 @@ impl ComInterface {
     pub fn from_type_name(name: TypeName) -> Self {
         let mut count = BTreeMap::new();
 
-        let methods = name
-            .def
-            .methods()
-            .map(|method| {
+        let mut bases = Vec::new();
```

Comment:
We probably want to use a `VecDeque` if you're going to be inserting at the beginning.

Reply:
Replaced with `push` and `rev` iterator.